geode-core/build.gradle

```diff
@@ -233,7 +233,9 @@ dependencies {
         exclude group: '*'
     }
 
-    runtimeOnly(project(':geode-deployment:geode-deployment-legacy'))
+    runtimeOnly(project(':geode-deployment:geode-deployment-legacy')) {
+        exclude module: 'geode-core'
```

Comment:
What is the actual problem being addressed by this change?

`geode-core` needs `geode-deployment:geode-deployment-legacy`, which needs `geode-core` to exist on the classpath. At runtime, this will be true. Why not declare it?

Reply:
You should be pairing with Dale and me then :) This was hard to debug.

I'm trying to back GEODE-9486. AnalyzeSerializables integration tests load the `sanctioned-geode-*-serializables.txt` resource and then generates a failure message to point the person running the test to overwrite the `sanctioned-geode-*-serializables.txt` with the generated actual serializables file. However, the error message was broken. After fixing it, it fails only in the modules that have these circular dependencies. We discovered that when the test runs (only in modules with circular dependencies) the classes and resources are loaded from the geode-*module*/build/libs/geode-*module*.jar which causes the failure message to blow up.

How about we pair from 10am to noon tomorrow?